bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -246,9 +286,19 @@ private void handlePlaybackCommands(Command command) {
      * - PUSH.STATE -
      */
     private void bindDefaultListener() {
-        volumio.on(Socket.EVENT_CONNECT, connectListener());
-        volumio.on(Socket.EVENT_DISCONNECT, disconnectListener());
-        volumio.on(VolumioEvents.PUSH_STATE, pushStateListener());
+        VolumioService volumioLocal = volumio;
+
+        if (volumioLocal == null) {
+            logger.debug("Volumio service was not yet initialized");
```

Comment:
Please remove logger and improve `ThingStatusDetail`

Reply:
solved